packages/website/src/pages/SearchResultPage.tsx

```diff
@@ -45,13 +46,14 @@ const ResultListRenderer: FunctionComponent<ResultListProps> = (props) => {
                             {state.results.length === 1 ? ' result' : ' results'}
                         </p>
                         <div className="tile-grid">
-                            {state.results.map(({title, raw, uniqueId, clickUri}: Result) => (
+                            {state.results.map((result) => (
                                 <Tile
-                                    key={uniqueId}
-                                    title={title}
-                                    href={clickUri.slice(clickUri.indexOf('#'))}
-                                    description={raw.description as string}
-                                    thumbnail={raw.thumbnail as TileProps['thumbnail']}
+                                    key={result.uniqueId}
+                                    title={result.title}
+                                    href={result.clickUri.slice(result.clickUri.indexOf('#'))}
+                                    description={result.raw.description as string}
+                                    thumbnail={result.raw.thumbnail as TileProps['thumbnail']}
+                                    sendAnalytics={() => engine.dispatch(logDocumentOpen(result))}
```

Comment:
Why is here a document open event, and the other ones are custom?

Reply:
This one is taking a Result as param and I don't have them in other places than the SearchResultPage. 
It makes me want to make the Home page just a big result page :see_no_evil: But that would still not work for the sidenav...